proof/refine/ARM/StateRelation.thy

```diff
@@ -456,7 +456,9 @@ definition
   sc_replies_relation_2 ::
   "(obj_ref \<rightharpoonup> obj_ref list) \<Rightarrow> (obj_ref \<rightharpoonup> obj_ref) \<Rightarrow> (obj_ref \<rightharpoonup> obj_ref) \<Rightarrow> bool" where
   "sc_replies_relation_2 sc_repls scRepl replPrevs \<equiv>
-     \<forall>p replies. sc_repls p = Some replies \<longrightarrow> heap_list replPrevs (scRepl p) replies"
+     \<forall>p replies. sc_repls p =
+            Some replies \<longrightarrow> heap_list replPrevs (scRepl p) replies
```

Comment:
Nitpick: I'm not a fan of splitting the `=` here - I was confused for a whole two seconds when I first saw this.